From b9cb23de4294ca7e675f2eca75bb7214b63b2a24 Mon Sep 17 00:00:00 2001
From: Salem Karani <salemkarani@gmail.com>
Date: Mon, 6 Oct 2014 08:27:26 -0400
Subject: [PATCH] French Patch

---
 HelloWorld/src/HelloWorld.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/HelloWorld/src/HelloWorld.java b/HelloWorld/src/HelloWorld.java
index b316ea1..e31d929 100644
--- a/HelloWorld/src/HelloWorld.java
+++ b/HelloWorld/src/HelloWorld.java
@@ -1,7 +1,7 @@
 
 public class HelloWorld {
 	public static void main( String [] args){
-		System.out.println("Hello World!");
+		System.out.println("Bonjour, monde");
 	}
 
 }
-- 
1.9.4

